Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.6.4-alpine3.9
+
+RUN apk add build-base postgresql-contrib postgresql-dev bash tzdata
+
+WORKDIR /app
+
+COPY Gemfile* .ruby-version ./
+
+RUN gem install bundler
+RUN bundle install --no-cache
+
+COPY . .
+
+ENV APP_PORT 3000
+EXPOSE $APP_PORT
+
+CMD bundle exec rake db:migrate && bundle exec rails s -p ${APP_PORT} --binding=0.0.0.0
```

Comment:
This is running as root. Which won't work with Kubernetes. We need a user > 1000. Have a look at the other dockerfiles for examples.

Reply:
OK, will have to make uid a ARG, As we are mounting the FS for faster test feedback we need to be able to configre what user is used (development needs the same uid to stop file permisions issues ect).

Reply:
latest commit should resolve this now